README.md

```diff
@@ -56,6 +56,7 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 - [vue-vben-admin-2.0](https://github.com/anncwb/vue-vben-admin) - Starter template with Vue 3, TypeScript, Tailwind CSS, Ant Design Vue2.
 - [electron-vue-next](https://github.com/ci010/electron-vue-next) - Starter template using Vue 3 in Vite and latest Electron with fluent VS Code debug and GitHub release process out-of-box.
 - [vite-components-playground](https://xxholly32.github.io/vite-components-playground/) - A common Vue 3 components playground.
+- [vite-electron-builder](https://github.com/cawa-93/vite-electron-builder/) - GitHub template for Electron apps. Used Vite for frontend and backend. Used electron-builder with auto-release and auto-updates for distribution.
```

Comment:
Can you shorten the description a bit? Something like that: 

```suggestion
- [vite-electron-builder](https://github.com/cawa-93/vite-electron-builder/) - GitHub template for Electron apps, with automatic releases and updates thanks to `electron-builder`.
```

Reply:
Is it possible to leave "Used Vite for frontend and backend"? In my opinion, this is exactly what makes my template different. While all the alternatives that I've seen use Vite only to build front and backend is built by rollup in my template Vite is used to build all parts of the application.

Reply:
How about this, then: 
> GitHub template for Electron apps using Vite for both back and front-end, with automatic releases and updates.